stages/org.osbuild.ostree.config

```diff
@@ -44,6 +44,10 @@ SCHEMA = """
             "type": "boolean"
           }
         }
+      },
+      "bls_append_except_default": {
+        "type": "boolean",
+        "description": "Enable support for GRUB password"
```

Comment:
this looks like it should at the same level as readonly and bootloader (under sysroot).